widgets/hero/styles/default.less

```diff
@@ -32,7 +33,9 @@
 			h1,h2,h3,h4,h5,h6{
 				line-height: 1.375em;
 				color: @heading_color;
-				text-shadow: 0 2px 2px rgba( 0, 0, 0, @heading_shadow/100 );
+				& when not ( @heading_shadow = 0 ) {
+					text-shadow: 0 2px 2px rgba( 0, 0, 0, @heading_shadow/100 );
```

Comment:
I recently made a change which allows float values in the slider. I.e. you can set min and max to 0 and 1 and step to 0.01

Reply:
I've adjusted the text shadow to reflect this change. Is there any sort of built-in migration code in the SiteOrigin Widgets Bundle? I ask as that'll make it easier to handle the transition between the current heading_shadow and the new value.

Reply:
Not purpose built for migrations, but we'd normally use the `modify_instance` function to make any required changes. Would just need to try test thoroughly to make sure it does the migration correctly in all cases. Perhaps less risky to leave `heading_shadow` as is... 🤔

Reply:
Ah. So that's what that does (or rather can do). I would rather leave this as is for the time being as there's no real harm in leaving it.

Reply:
Ok, awesome. Merging.